cyoa/app/controllers/games_controller.rb

```diff
@@ -0,0 +1,13 @@
+class GamesController < ApplicationController
+  def create
+    @game = Game.create
+    @game.create_rounds
+    current_user.games << @game
```

Comment:
Ideally, models are not saved in an intermediate state where it gets confused. If the request fails on `@game.create_rounds` (timeout, server outage, etc), you have half a game built and end up with garbage data.

In this case, you can create a helper method that generates a new game for the user (say, `Game.generate(current_user)`), and it will not save until the last step.